tests/preprocessing/test_data_preprocessor.py

```
@@ -0,0 +1,116 @@
+import pandas as pd
+import numpy as np
+import pytest
+from power_consumption.preprocessing.data_preprocessor import DataProcessor
+from power_consumption.config import Config
```

Ruff

`power_consumption.config.Config` imported but unused

Remove unused import: `power_consumption.config.Config`

(F401)